jobdispatcher/src/main/java/com/firebase/jobdispatcher/BundleProtocol.java

```diff
@@ -44,5 +43,7 @@
   static final String PACKED_PARAM_TRIGGERED_URIS = "triggered_uris";
   static final String PACKED_PARAM_OBSERVED_URI = "observed_uris";
 
-  BundleProtocol() {}
+  private BundleProtocol() {
+     throw new AssertionError("No instance for you!");
+  }
```

Comment:
Are the AssertionErrors necessary for Kotlin? Otherwise I'd prefer to just keep the constructors empty (private is fine tho)

Reply:
sgtm